src/middleware/sentry.js

```diff
@@ -1,6 +1,7 @@
 const Sentry = require('@sentry/node');
 const { sha } = require('../helper/version');
 const { version } = require('../../package.json');
+const { deepObjectProps } = require('../utils/');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['deepObjectProps' is assigned a value but never used. (no-unused-vars)](https://app.codacy.com/app/schul-cloud/schulcloud-server/pullRequest?prid=4646529)